energy-integration/governance-v2/meta/Cargo.toml

```diff
@@ -11,4 +11,4 @@ authors = [ "you",]
 path = ".."
 
 [dependencies.elrond-wasm-debug]
-version = "=0.36.1"
+version = "0.36.1"
```

Comment:
Leave the `=0.36.1` please, this ensures we use that specific version.

Reply:
done